CMakeLists.txt

```diff
@@ -30,16 +30,19 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -m64 -O3 -funroll-loops -pipe")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -mmmx -mavx -msse -msse3")
-
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wsign-compare")
+if(UNIX) 
+  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -m64 -O3 -funroll-loops -pipe")
+  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -mmmx -mavx -msse -msse3")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wsign-compare")
+else(UNIX)
```

Comment:
should this be a regular `else()`?

Reply:
Cmake allows arguments to else, endif etc as fiducial labels (used to require them).  If given they must match the argument of matching bracket.  So this is OK, but not necessary.
I see now that it actually is my fault that the Linux build failed with"could not find zlib".  Evidently Linux packages don't need an external zlib; but Windows packages universally do, so I just reflexively added find_package(zlib).  In the latest commit, which I will push soon, this line is protected by if(WIN32) so that build error should not happen again.
The new cmakelists includes the capability to create a vcpkg instance and populate it with all the required packages (except ICPC, which is not on the vcpkg repo); or alternatively to use an existing vcpkg instance.  This is supported by cmake scripts in a new directory 'build_win32', which also has instructions for my windows build.  All of this is WIN32-only and should have no effect on the Linux build.